### PR TITLE
fix #3647 - clarify positional parameters 

### LINKED ISSUE #3647: parameter position
I think the following is incorrect:
    'If set to "0" or "named", a parameter name is required.'

For example, the Get-History cmdlet has 2 optional parameters (-Id and -Count). -Id has position 0 and -Count has position 1. If the above sentence was correct, -Id parameter name would be required but it is not.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3986212c-97f0-54e0-8954-a1e109230b58
* Version Independent ID: 098d2358-c416-c868-9b6c-02bc605f91be
* Content: [about_Parameters](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_parameters?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md)
* Product: **powershell**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d0c5fb5](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/d0c5fb5001a09cebf852f5d94c1a11bccc016db7)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d0c5fb5](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/d0c5fb5001a09cebf852f5d94c1a11bccc016db7)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d0c5fb5](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/d0c5fb5001a09cebf852f5d94c1a11bccc016db7)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-3.0&branch=pr-en-us-3653) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-4.0&branch=pr-en-us-3653) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.0&branch=pr-en-us-3653) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.1&branch=pr-en-us-3653) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-6&branch=pr-en-us-3653) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902051555100858-3653%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kiQqXOj6AROOx2OraE9qx7ZEdu24YLn22SYL%2FSnV3i0%3D&st=2019-02-05T16%3A04%3A05Z&se=2019-03-08T16%3A09%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[738911a](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/738911a03480f1700cac5b419cb448ee0377310b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[738911a](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/738911a03480f1700cac5b419cb448ee0377310b)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md [hunk 1]
```diff
@@ -102,24 +101,19 @@ parameter attributes.
 
 This setting indicates whether the parameter is mandatory, that is, whether
 all commands that use this cmdlet must include this parameter. When the value
-is **"True"** and the parameter is missing from the command, Windows PowerShell
+is **"True"** and the parameter is missing from the command, PowerShell
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md]: If it's being bolded, should it also have quotations?

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md [hunk 2]
```diff
@@ -102,24 +101,19 @@ parameter attributes.
...
 
-If the `Parameter position` setting is set to an integer other than **"0"**, the
-parameter name is not required. This type of parameter is referred to as a
-positional parameter, and the number indicates the position in which the
-parameter must appear in relation to other positional parameters. If you
-include the parameter name for a positional parameter, the parameter can be
-listed in any position after the cmdlet name.
-
 For example, the `Get-ChildItem` cmdlet has Path and Exclude parameters. The
 `Parameter position` setting for `Path` is **"1"**, which means that it is a
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md]: If changing them all from `parameter position` to `position`, missed this one.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[738911a](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/738911a03480f1700cac5b419cb448ee0377310b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-3.0&branch=pr-en-us-3653) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-4.0&branch=pr-en-us-3653) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.0&branch=pr-en-us-3653) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.1&branch=pr-en-us-3653) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-6&branch=pr-en-us-3653) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902051604540012-3653%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OGjQsVr5IQmW9TjEG80ZNCFu13sZiaWDYT6wnozE4vY%3D&st=2019-02-05T16%3A17%3A43Z&se=2019-03-08T16%3A22%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md (hunk 2 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md]: Fixed. And I found another thing that needed changing.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[7ef286b](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/7ef286b6a47bf2bd88986f1733f01ec63055d5b6)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[7ef286b](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/7ef286b6a47bf2bd88986f1733f01ec63055d5b6)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[7ef286b](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/7ef286b6a47bf2bd88986f1733f01ec63055d5b6)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-3.0&branch=pr-en-us-3653) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-4.0&branch=pr-en-us-3653) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.0&branch=pr-en-us-3653) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.1&branch=pr-en-us-3653) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-6&branch=pr-en-us-3653) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902051641507264-3653%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5OW6ybe8XxrgtjLUPguoTjQXKZBRPuLb927S6d3oKHA%3D&st=2019-02-05T16%3A51%3A01Z&se=2019-03-08T16%3A56%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2359419](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/235941970c16a5e38bbcd86f04ff34c58bd4259b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2359419](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/235941970c16a5e38bbcd86f04ff34c58bd4259b)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2359419](https://github.com/sdwheeler/PowerShell-Docs/pull/3653/commits/235941970c16a5e38bbcd86f04ff34c58bd4259b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-3.0&branch=pr-en-us-3653) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-4.0&branch=pr-en-us-3653) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.0&branch=pr-en-us-3653) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-5.1&branch=pr-en-us-3653) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_parameters?view=powershell-6&branch=pr-en-us-3653) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/4.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.0/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/5.1/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3647/reference/6/Microsoft.PowerShell.Core/About/about_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902051655296726-3653%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=aa%2FDTV%2BeS9lIv6KdXYIQLc1qHQqDKn7r51PzbbWR6ck%3D&st=2019-02-05T17%3A05%3A57Z&se=2019-03-08T17%3A10%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**